### PR TITLE
Remove unnecessary casts and TODO's

### PR DESCRIPTION
**Pull request by TimvdLippe**
I found these TODO's while looking through the code. Since the corresponding PR has since been closed and the typings of Node are updated, the casts can be removed.

 - [x] CHANGELOG.md has not been updated, internal NOOP change

### CODE REVIEW

File: src/html-splitter.ts [hunk 1]
```diff
@@ -102,14 +102,17 @@ export class SplitFile {
...
+    console.assert(
+        this.parts.get(path) !== undefined,
+        `Trying to save unexpected file part "${path}".`);
+    console.assert(
+        this.parts.get(path) === null,
+        `Trying to save already-saved file part "${path}".`);
+    console.assert(
+        this.outstandingPartCount > 0,
+        `Trying to save valid file part "${path}", ` +
+            `but somehow no file parts are outstanding.`);
```
Comment (stramel) [inline src/html-splitter.ts]: Can't template literal strings span multiple lines without the need for concatenation

**Comment by stramel** [comment]
Dang you beat me to these! Meant to throw out a PR when I was working on the multiple builds for custom-build template

File: src/html-splitter.ts (hunk 1 above)
Comment (TimvdLippe) [inline src/html-splitter.ts]: That would insert new lines instead and in this case, we want to assert that the single line is a warning.

**Comment by TimvdLippe** [comment]
Thanks for the speedy reviews!

**Pull request merged**